client-src/elements/chromedash-ot-creation-page.js

```diff
@@ -152,7 +152,7 @@ export class ChromedashOTCreationPage extends LitElement {
 
   renderFields(section) {
     const fields = section.fields.map(field => {
-      const value = getStageValue(this.stage, field);
+      let value = getStageValue(this.stage, field);
```

Comment:
It's odd that this did not cause an error at build time.

Reply:
I think this might also be from a git issue we spoke of before. This is causing runtime problems in the code now, but in a place that is not accessible to users. 😅